### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by eli7890**
View of metadata

### CODE REVIEW

File: bin/main/templates/docs/docs.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                        </select>
+                    </div>
+                    <div class="col-md-4 mb-3">
+                        <label for="language">Format</label>
+                        <select class="custom-select" id="format"  name = "language" required th:field="*{format}">
+                            <option th:value="'screenShot'" th:text="ScreenShot"></option>
+                            <option th:value="'txt'" th:text="Txt"></option>
+                            <option th:value="'docx'" th:text="Docx"></option>
+                            <option th:value="'pdf'" th:text="PDF"></option>
+
```
Comment (roht0515) [inline bin/main/templates/docs/docs.html:37]: remove this blank space please

File: src/main/resources/static/css/style.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+#galeria .container-fluid{
+    padding-left: 0;
+    padding-right: 0;
+}
+.cuadrado-perfecto{
+    width: 100%;
+    padding-bottom: 100%;
+    position: relative;
+    /*border: 1px solid #ccc;*/
```
Comment (roht0515) [inline src/main/resources/static/css/style.css:61]: delete this line if you are not going to use it

File: src/main/resources/static/css/style.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    .section{
+        padding-top: 30px;
+        padding-bottom: 30px;
+    }
+    #galeria .columna{
+        width: 100%;
+    }
+}
+@media(min-width: 992px) and (max-width: 1199px){
+
```
Comment (roht0515) [inline src/main/resources/static/css/style.css:93]: remove this blank space please

**Review by julia-jpg — APPROVED** [review]

File: bin/main/templates/main/template.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</header>
+<!-- CONTENT-->
+<div class="container">
+
+</div>
```
Comment (roht0515) [inline bin/main/templates/main/template.html:48]: <div class="container"></div>
can you change it like this?

File: src/main/resources/static/css/style.css [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+}
+#galeria .columna{
+    padding-right: 0;
+    padding-left: 0;
+    width: 25%
+}
+#galeria .row{
+    padding-right: 0;
+    padding-left: 0;
+    margin-right:0;
```
Comment (roht0515) [inline src/main/resources/static/css/style.css:38]: add a blank space after ":" like this margin-right: 0;

**Review by julia-jpg — APPROVED** [review]

**Pull request merged**